src/helpers/counters/Counter.jsx

```diff
@@ -0,0 +1,14 @@
+import "./Counter.css"
+
+function Counter(props) {
+    return (
+        <div className="fruit-counter">
+            <h1>{props.title}</h1>
+            <button disabled={props.count < 1} onClick={props.onDecrement}>-</button>
```

Comment:
Ik zou je als tip mee willen geven om na te denken over wanneer je `on` gebruikt in functienamen. `on` wordt vaak toegepast als de functie uitgevoerd moet worden ná de actie. Dus in dit geval: "als de decrement heeft plaats gevonden dan voer ik de onDecrement functie uit". Alleen hier wil je juist de actie zélf uitvoeren. Mijn tip zou dan ook zijn:
```suggestion
            <button disabled={props.count < 1} onClick={props.decrement}>-</button>
```